3-reveal/migrations/5-IRS/4-Zambia-2020/deploy/zambia_irs_mop_up.psql

```diff
@@ -39,8 +61,8 @@ SELECT DISTINCT ON (locations.id, events_query.task_id)
     COALESCE(events_query.notsprayed_reasons, ARRAY[]::varchar[]) AS notsprayed_reasons,
     COALESCE(events_query.duplicate, FALSE) AS duplicate,
     COALESCE(events_query.sprayed_duplicate, FALSE) AS sprayed_duplicate
-FROM locations
-INNER JOIN zambia_structure_jurisdictions ON zambia_structure_jurisdictions.id = locations.id
+FROM pending.restricted_structures AS structures
```

Comment:
Do you mind giving this a "locations" alias?  It would then be simpler to change this at just one place in the code when we want to deploy it to the new  database structure that calls structures "locations"

Reply:
oh, right.  Seemed off and didn't realize it was happening yet